src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double calculateArea() {
+        return Math.PI * Math.pow(radius, 2);
+    }
+
+    @Override
+    public String display() {
+        return "Figure: circle, area " + String.format("%.2f", calculateArea())
+                + " sq. units, radius " + radius + " units, color: " + getColor();

```

Comment:
There is a bad solution for String concatenation with + operator. Read [this](https://github.com/mate-academy/jv-oop-advanced/pull/1273/javatpoint.com/java-string-format) article please.